tests/test_multichannel.py

```diff
@@ -159,6 +163,250 @@ def test_griffinlim_cqt_multi(y_multi):
+    assert np.allclose(C1, Call[1])
+
+    # Verify that they're not all the same
+    assert not np.allclose(Call[0], Call[1])
+
+def test_spectral_rolloff_multi_variable(s_multi):
+    S, sr = s_multi
+
+    freq = np.random.randn(*S.shape)
+
+    # compare each channel
+    C0 = librosa.feature.spectral_rolloff(sr=sr, freq=freq[0], S=S[0])
+    C1 = librosa.feature.spectral_rolloff(sr=sr, freq=freq[1], S=S[1])
+    Call = librosa.feature.spectral_rolloff(sr=sr, freq=freq, S=S)
+
+    # Check each channel
+    assert np.allclose(C0, Call[0])
+    assert np.allclose(C1, Call[1])
+
+    # Verify that they're not all the same
+    assert not np.allclose(Call[0], Call[1])
+
+def test_spectral_flatness_multi(s_multi):
+    S, sr = s_multi
+
+    # compare each channel
+    C0 = librosa.feature.spectral_flatness(S=S[0])
+    C1 = librosa.feature.spectral_flatness(S=S[1])
+    Call = librosa.feature.spectral_flatness(S=S)
+
+    # Check each channel
+    assert np.allclose(C0, Call[0],atol=1e-5)
+    assert np.allclose(C1, Call[1])
+
+    # Verify that they're not all the same
+    assert not np.allclose(Call[0], Call[1])
+
+def test_rms_multi(s_multi):
+    S, sr = s_multi
+
+    # compare each channel
+    C0 = librosa.feature.rms(S=S[0])
+    C1 = librosa.feature.rms(S=S[1])
+    Call = librosa.feature.rms(S=S)
+
+    # Check each channel
+    assert np.allclose(C0, Call[0])
+    assert np.allclose(C1, Call[1])
+
+    # Verify that they're not all the same
+    assert not np.allclose(Call[0], Call[1])
+
+def test_zcr_multi(y_multi):
+    y, sr = y_multi
+
+    # compare each channel
+    C0 = librosa.feature.zero_crossing_rate(y=y[0])
+    C1 = librosa.feature.zero_crossing_rate(y=y[1])
+    Call = librosa.feature.zero_crossing_rate(y=y)
+
+    # Check each channel
+    assert np.allclose(C0, Call[0])
+    assert np.allclose(C1, Call[1])
+
+    # Verify that they're not all the same
+    assert not np.allclose(Call[0], Call[1])
+
+def test_chroma_stft_multi(s_multi):
+    S, sr = s_multi
+
+    # compare each channel
+    C0 = librosa.feature.chroma_stft(S=S[0],tuning=0)
+    C1 = librosa.feature.chroma_stft(S=S[1],tuning=0)
+    Call = librosa.feature.chroma_stft(S=S,tuning=0)
+
+    # Check each channel
+    assert np.allclose(C0, Call[0])
+    assert np.allclose(C1, Call[1])
+
+    # Verify that they're not all the same
+    assert not np.allclose(Call[0], Call[1])
+
+def test_mfcc_multi(s_multi):
+    S, sr = s_multi
+
+    # compare each channel
+    C0 = librosa.feature.mfcc(S=librosa.core.amplitude_to_db(S=S[0],top_db=None))
+    C1 = librosa.feature.mfcc(S=librosa.core.amplitude_to_db(S=S[1],top_db=None))
+    Call = librosa.feature.mfcc(S=librosa.core.amplitude_to_db(S=S,top_db=None))
+
+    # Check each channel
+    assert np.allclose(C0, Call[0])
+    assert np.allclose(C1, Call[1])
+
+    # Verify that they're not all the same
+    assert not np.allclose(Call[0], Call[1])
+
+def test_mfcc_multi_time(y_multi):
```

Comment:
This one fails because of the power_to_db calculation in mfcc when the input spectrogram is not provided.
For now, I think we can bypass this test.  Rather than delete it, let's add `@pytest.mark.skip(reason='power_to_db leaks information across channels')` on this test, and then we can come back and fix it later (if we provide API hooks that get us around it).